index.html

```diff
@@ -292,7 +293,7 @@
 <section id="resources-included">
 <h3>Resources Included in the <a>PerformanceResourceTiming</a> Interface</h3>
 
-<p>All resources <a title='fetch' href="https://www.w3.org/TR/html5/infrastructure.html#fetch">fetched</a> by the current <a href="https://www.w3.org/TR/html5/browsers.html#browsing-context">browsing</a> [[!HTML5]] or worker [[!WORKERS]] context's MUST be included as <a>PerformanceResourceTiming</a> objects in the <a href="https://www.w3.org/TR/performance-timeline-2/#sec-performance-timeline">Performance Timeline</a> of the relevant context. Resources that are retrieved from <a href="https://www.w3.org/TR/html5/browsers.html#relevant-application-cache">relevant application caches</a> or local resources MUST be included as <a>PerformanceResourceTiming</a> objects in the <a href="https://www.w3.org/TR/performance-timeline-2/#sec-performance-timeline">Performance Timeline</a> [[!PERFORMANCE-TIMELINE-2]]. Resources for which the <a title='fetch' href="https://www.w3.org/TR/html5/infrastructure.html#fetch">fetch</a> was initiated, but was later aborted (e.g. due to a network error) MUST be included as <a>PerformanceResourceTiming</a> objects in the <a href="https://www.w3.org/TR/performance-timeline-2/#sec-performance-timeline">Performance Timeline</a> and MUST contain initialized attribute values for processed substeps of the <a href="#processing-model">processing model</a>.</p>

```

Comment:
Note that this will undo some of the L3-specific work we merged earlier, which updated the requirements from ~"UA MAY include aborted fetches" to "MUST".

Stepping back, I think that may actually be the only major difference between current L2 and L3 specs. Also, given that we're planning to rewrite L3 in terms of Fetch.. the L3 update is something we'll have to do redo anyway -- originally we thought we'd merge this into L2, but at TPAC we decided to punt this requirement to L3. 

Long story short, merging this will undo some of the L3 work, but I think that may be OK.. In the meantime, it'll make our life a lot easier to keep v2 and gh-pages in sync. Once we're ready to start working on L3 in earnest, we'll sort out the MAY/MUST bits. 

@toddreifsteck wdyt?